ethcore/src/account_db.rs

```diff
@@ -78,6 +78,7 @@ pub struct AccountDB<'db> {
 
 impl<'db> AccountDB<'db> {
 	/// Create a new AccountDB from an address.
+	#[allow(dead_code)]
 	pub fn new(db: &'db HashDB, address: &Address) -> Self {
```

Comment:
There are other constructors here which were marked in the same way. I'm not opposed to removing.